internal/result/results.go

```diff
@@ -51,26 +51,26 @@ var (
 	}
 )
 
-// LTE returns if rowResult is less than or equal to compareTo.
-func LTE(rowResult, compareTo statuspb.TestStatus) bool {
+// lte returns if rowResult is less than or equal to compareTo.
+func lte(rowResult, compareTo statuspb.TestStatus) bool {
 	return statusSeverity[rowResult] <= statusSeverity[compareTo]
 }
 
-// GTE returns if rowResult is greater than or equal to compareTo.
-func GTE(rowResult, compareTo statuspb.TestStatus) bool {
```

Comment:
Can we keep this exported for internal use, or should we just have it exported there?

Reply:
It wasn't exported historically? Does something need to use it internally?